packages/console/src/components/Executions/ExecutionDetails/utils.ts

```diff
@@ -27,10 +29,22 @@ export function getExecutionBackLink(execution: Execution): string {
 export function getTaskExecutionDetailReasons(
   taskExecutionDetails?: PaginatedEntityResponse<TaskExecution>,
 ): (string | null | undefined)[] {
-  return (
+  const reasons = (
     taskExecutionDetails?.entities.map(
-      taskExecution => taskExecution.closure.reason,
+      taskExecution => taskExecution.closure.reasons || [],
```

Comment:
We need to add a fallback here for cases where users are running older versions of admin; fall back to `reason`.